stage-1-proposals.md

```diff
@@ -224,7 +224,7 @@ See also the [active proposals](README.md), [stage 0 proposals](stage-0-proposal
 [double-ended-iterator-notes]: https://github.com/tc39/notes/blob/HEAD/meetings/2020-09/sept-24.md#double-ended-iterator-and-destructuring-for-stage-1
 [debug]: https://github.com/tc39/proposal-standardized-debug
 [debug-notes]: https://github.com/tc39/notes/blob/HEAD/meetings/2020-11/nov-17.md#standardized-debug-for-stage-2
-[modulus]: https://github.com/phoddie/integer-and-modulus-math-proposal
+[modulus]: https://github.com/tc39/integer-and-modulus-math-proposal
```

Comment:
suggest, rename the proposal repo name. (move `proposal` to prefix)

Reply:
good call, done